runtime/doc/lsp.txt

```diff
@@ -1782,14 +1789,19 @@ hover({config})                                          *vim.lsp.buf.hover()*
 <
 
     Parameters: ~
-      • {config}  (`vim.lsp.buf.hover.Opts?`) See |vim.lsp.buf.hover.Opts|.
+      • {config}    (`vim.lsp.buf.hover.Opts?`) See |vim.lsp.buf.hover.Opts|.
+      • {callback}  (`fun(errors: lsp.ResponseError[])?`) Function called
+                    after the request is completed. Any errors are passed as
+                    the first argument.
```

Comment:
This is the only case that mentions "Any errors are passed...". Maybe worth defining a private `@class` to define the function signature, and use `@inlinedoc` to repeat the docs?

Reply:
My Lua expertise is lacking: How could I annotate a function with `@class`?

Reply:
I tried that but then `make doc` types `fn` as `MyFunction` without inlining the function signature.

Reply:
@lewis6991 what are your thoughts on having an `lsp.Callback` alias for this function (similar to `lsp.Handler`)?